src/schema.ts

```diff
@@ -1090,3 +1090,37 @@ export const listPostRelations = relations(listPosts, ({ one }) => ({
     references: [posts.id],
   }),
 }));
+
+export const relayStateEnum = pgEnum("relay_state", [
+  "idle",
+  "pending",
+  "accepted",
+  "rejected",
+]);
+
+export const relays = pgTable("relays", {
+  relayServerActorId: uuid("relay_server_actor_id")
+    .$type<Uuid>()
+    .references(() => accounts.id, { onDelete: "cascade" }),
+  state: relayStateEnum("state").notNull().default("idle"),
+  followRequestId: text("follow_request_id").notNull().unique(),
```

Comment:
Shouldn't this reference a sent follow request? Though maybe we're not storing in the database the follow requests we've sent? cc @dahlia

Reply:
Looks like we could actually store the follow request, by doing an insert to `follows` for the relay actor, and having approved be null?

From: https://github.com/dahlia/hollo/blob/9d5aa116141e43ed824ea8b44ba621f5978143c8/src/api/v1/follow_requests.ts

```
const followers = await db.query.follows.findMany({
    where: and(eq(follows.followingId, owner.id), isNull(follows.approved)),
    with: { follower: { with: { owner: true, successor: true } } },
  });
```

Reply:
Yeah, and as the primary key of the `follows` table is `iri`, the column should be renamed to `followRequestIri`:

```suggestion
  followRequestIri: text("follow_request_iri")
    .notNull()
    .unique()
    .references(() => follows.iri, { onDelete: "cascade" }),
```